vocab/unit/VOCAB_QUDT-UNITS-ALL-v2.1.ttl

```diff
@@ -7505,6 +8603,15 @@ unit:FT_H2O
   rdfs:isDefinedBy <http://qudt.org/2.1/vocab/unit> ;
   rdfs:label "Foot of Water"@en ;
 .
+<http://qudt.org/vocab/unit/FT_H2O_39.2DEG_F>
```

Comment:
The `'.'` here trips up my generation of Java/Javascript constants (no dots allowed). Not a huge deal, but I noticed QUDT already has a rule for commas in dimension vectors: replace `.` with `'pt'`. It would be consistent to apply this here, too.

Reply:
So, the suggestion would be to use `FT_H2O_39pt2DEG_F`

Reply:
@steveraysteveray is it ok if I make that change in my upcoming commit(s) to this branch?